fuzz/fuzz_targets/mutate.rs

```diff
@@ -0,0 +1,70 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+use std::sync::atomic::{AtomicU64, Ordering};
+use wasmparser::WasmFeatures;
+
+static NUM_RUNS: AtomicU64 = AtomicU64::new(0);
+static NUM_SUCCESSFUL_MUTATIONS: AtomicU64 = AtomicU64::new(0);
+
+fuzz_target!(|inputs: (wasm_smith::Module, u64)| {
```

Comment:
Could this use https://github.com/bytecodealliance/wasm-tools/blob/9130f0f0233fe44626368e91579529e646080820/fuzz/src/lib.rs#L5-L8 perhaps using the closure to generate the seed?